config/config.go

```diff
@@ -30,6 +30,9 @@ const (
 	// Ubuntu18 is Ubuntu Bionic
 	Ubuntu18 = "bionic"
 
+	// Ubuntu 19 is Disco Dingo
```

Comment:
__[Golint]__ comment on exported const Ubuntu19 should be of the form "Ubuntu19 ..." ([link](https://sider.review/gh/repos/87515828/pulls/64#issue-4103535))

<sub>posted by Sider</sub>

Reply:
Fixed now.